DotNet/parser.js

```diff
@@ -10,22 +10,30 @@
 	}
 	let element = ''
 	let options = ''
+	let scrollImplemented = false
 	if (cs.frames === undefined && cs.region === undefined) element = '.Window()'
 	else {
-		if (cs.frames) element += frames(cs.frames)
+		if (cs.frames) {
+			if (cs.scrollRootElement) {
+				element += `.Window()` + scrollRootElement(cs.scrollRootElement)
+				scrollImplemented = true
+			}
+			element += frames(cs.frames)
+		}
 		if (cs.region) {
 			if (cs.region.type) element += region(cs.region, cs.region.type, mobile)
 			else element += region(cs.region, undefined, mobile)
 		}
 	}
 	if (cs.ignoreRegions) options += ignoreRegions(cs.ignoreRegions, mobile)
 	if (cs.floatingRegions) options += floatingRegions(cs.floatingRegions)
-	if (cs.scrollRootElement) options += scrollRootElement(cs.scrollRootElement)
+	if ((cs.scrollRootElement) && (!scrollImplemented)) options += scrollRootElement(cs.scrollRootElement)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
	if ((cs.scrollRootElement) && (!scrollImplemented)) {options += scrollRootElement(cs.scrollRootElement)}
```